styles/styles.css

```diff
@@ -257,3 +257,34 @@ main .section.highlight {
   margin: 0;
   padding: 40px 0;
 }
+
+.blue-section {
+    padding: 20px;
+    background-color: #388ce1;
+    color: white;
+    text-align: center;
+    & h4 {
+        font-family: franklin-heavy-italic, sans-serif;
```

Comment:
This should reference a variable - not sure where `franklin-heavy-italic` is defined.

Should have a variable
```
--font-family-open-sans: 'Open Sans', 'Open Sans Fallback', Arial, sans-serif
```

Which requires a fallback definiton

Reply:
For this one Alex has a PR that solves fonts issues and will use the families defined in there.